data/net/minecraft/data/worldgen/placement/VegetationPlacements.mapping

```diff
@@ -0,0 +1,11 @@
+CLASS net/minecraft/data/worldgen/placement/VegetationPlacements
```

Comment:
For all the parameters in this class, I think `placement` is a better name than `modifier`, given it's more descriptive of the actual name, and matches the mojang name better.

Reply:
Parameter names changes. All suggestions have been applied now.